lib/morpheus.gi

```diff
@@ -2374,8 +2374,20 @@ InstallGlobalFunction(AutomorphismGroupFittingFree,function(g)
 
   aup:=Normalizer(d,Image(emb,g));
 
-  #reduce degree
-  s:=SmallerDegreePermutationRepresentation(aup:cheap);
+  #reduce degree -- avoid redundatnt fixedpoints
+  if count>1 then
+    p:=aup;
+    s:=IdentityMapping(p);
+    repeat
+      i:=SmallerDegreePermutationRepresentation(p);
+      if NrMovedPoints(Image(i))<NrMovedPoints(p) then
+        p:=Image(i,p);
```

Comment:
Perhaps like this for consistency?
```suggestion
        p:=Image(i);
```
Could also use `ImagesSource(i)` here and in the line above

Reply:
The `Image(i,p)` was deliberate, to ensure that -- even if the homomorphism is defined on a larger set, which I cannot prove will never happen -- the image group is the same. But the first `Image` could be the range.